src/Apps/Order/Routes/__tests__/Review.test.tsx

```diff
@@ -43,19 +43,8 @@ describe("Review", () => {
     )
   }
 
-  it("disables the button while the terms are unchecked", () => {
-    const component = getWrapper(defaultProps)
-    expect(component.find(Button).props().disabled).toBe(true)
-    expect(pushMock).not.toBeCalled
-  })
-
   it("enables the button while the terms are checked and routes to the payoff page", () => {
```

Comment:
#minor but this test name no longer applies! We should update to something like `"routes to the payoff page when the button is clicked"